experimental/rocm/native_executable.h

```diff
@@ -14,6 +14,9 @@
 #include "iree/base/api.h"
 #include "iree/hal/api.h"
 
+// flatcc schemas:
+#include "iree/schemas/rocm_executable_def_reader.h"
```

Comment:
we don't really want to expose the flatbuffer internals (as they may change, and we may have different versions) - we should store whatever we need during execution on the params struct instead of the source of it